Exercise_7/Program.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.Serialization.Json;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+using System.Xml.Linq;
+
+namespace Exercise_7
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            ConvertJsonToXml("bus-services.json", "bus-services.xml");
```

Comment:
Create a list and call. `files.Foreach(ConvertJsonToXml)` where `ConvertJsonToXml` is `static void ConvertJsonToXml(string fileName)` and fileName without `.json`

Reply:
Done